stable/kong/values.yaml

```diff
@@ -384,3 +384,30 @@ ingressController:
     name:
 
   ingressClass: kong
+
+# We pass the declarative config over here.
+declarativeMode:
+  # Either Kong's configuration is managed from an existing ConfigMap
+  existingConfigMap: ""
```

Comment:
Let's keep the name of this property as `configMap`.
The comment above it will help the users on how to use it.

Reply:
Changing to the new value. It is however common to use the 'existing' prefix whenever a helm  is referencing to existing secrets or configMaps.....